.circleci/config.yml

```diff
@@ -303,6 +303,33 @@ jobs:
           command: DD_TRACE_AGENT_TIMEOUT=2000 composer << parameters.integration_testsuite >>
       - <<: *STEP_PERSIST_TO_WORKSPACE
       - <<: *STEP_STORE_TEST_RESULTS
+  hyperfine_tests:
+    working_directory: ~/datadog
+    parameters:
+      docker_image:
+        type: string
+      lib_curl_command:
+        type: string
+    executor:
+      name: with_agent
+      docker_image: << parameters.docker_image >>
+    steps:
+      - <<: *STEP_ATTACH_WORKSPACE
+      - lib_curl_workaround:
+          command: << parameters.lib_curl_command >>
+      - prepare_extension_and_composer_with_cache
+      - <<: *STEP_PREPARE_TEST_RESULTS_DIR
+      - <<: *STEP_EXPORT_CI_ENV
+      - <<: *STEP_WAIT_AGENT
+      - run: |
+          set -xe
+          curl -q -o hyperfine.tgz -L https://github.com/sharkdp/hyperfine/releases/download/v1.6.0/hyperfine-v1.6.0-x86_64-unknown-linux-musl.tar.gz
+          tar xf hyperfine.tgz --strip-component=1
+          php -m | grep ddtrace
+          sudo rm /usr/local/etc/php/conf.d/ddtrace.ini
+          php -m | grep ddtrace || true
+          ./hyperfine -s basic 'php -m' 'php -m -dextension=ddtrace.so' --min-runs 10000 --export-markdown hyperfine.md
```

Comment:
It would be interesting to see if we could run these with **tests/ext/dd_trace_forward_call_for_functions.phpt** or some other script. And for the second command, we could add `-dddtrace.request_init_hook=<path/to>/bridge/dd_wrap_autoloader.php` to include the `request_init_hook` in the measurement.

This would be an improvement in a separate PR, but I love that you added this! :)

Reply:
Definately agree - lets  expand on this in more PRs!